modules/API/API.js

```diff
@@ -1004,7 +993,13 @@ function initCommands() {
             callback(isP2pActive(APP.store.getState()));
             break;
         }
+        case '_is_electron_screensharing': {
+            callback(true);
+            break;
+        }
         default:
+            callback(null);
```

Comment:
Maybe we should use an error object here. With some specific name like `UnknownRequestError`.